src/components/ServiceDetails/AboutService/DemoExample/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import { withStyles } from "@material-ui/styles";
+
+import ProgressBar from "../../../common/ProgressBar";
+import { useStyles } from "./styles";
+import ServiceProvider from "./ServiceProvider";
+
+class DemoExample extends Component {
+  state = {
+    activeSection: 1,
+    error: "error state message",
+    progressText: ["Configure", "Purchase", "Results"],
+    protoSpec: undefined,
+    serviceSpecJSON: undefined,
+    grpcResponse: undefined,
+  };
+
+  render() {
+    const { classes, service } = this.props;
+    const { activeSection, progressText } = this.state;
+    return (
+      <div className={classes.demoExampleContainer}>
+        <h4>Process</h4>
+        <ProgressBar activeSection={activeSection} progressText={progressText} />
```

Comment:
`ServiceDemoSteps` would be more suitable than `ProgressBar`.

Reply:
[renamed](https://github.com/singnet/refactored-dapp/pull/13/commits/ea4add782480235c55f8ca04d7fcab3065fe31fd)

Reply:
ProgressBar is a common component..

Reply:
In which I propose renaming it since it could lead to confusion. Something like `ProcessSteps` or `ProcessProgress`. Feel free to pick something more relevant to other places where it is being used.